Formula/graph-tool.rb

```diff
@@ -32,29 +32,31 @@ class GraphTool < Formula
   depends_on "librsvg"
   depends_on macos: :mojave # for C++17
   depends_on "numpy"
+  depends_on "pillow"
   depends_on "py3cairo"
   depends_on "pygobject3"
   depends_on "python@3.9"
   depends_on "scipy"
+  depends_on "six"
```

Comment:
Are these two requirements new?

Reply:
`six` has been removed as a resource and made a dependency, I'm unsure about `pillow`.

Reply:
pillow was listed when I did the python dependencies check, let me see if i can run without it.

Reply:
> `six` has been removed as a resource and made a dependency, I'm unsure about `pillow`.

this is the common practice we are doing right now.

Reply:
Installing `graph-tool` using `conda` installs `pillow` though, so it might be needed at runtime at some point